src/components/Founder.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect } from 'preact/hooks'
+import Description from 'components/Description'
+import KetlLogo from 'icons/KetlLogo'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  flexDirection,
+  fontSize,
+  gap,
+  justifyContent,
+  opacity,
+  padding,
+  space,
+  textAlign,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+import createFounderProof from 'helpers/createFounderProof'
+import postWebViewMessage from 'helpers/postWebViewMessage'
+import Messages from 'models/Messages'
+
+const container = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  justifyContent('justify-center'),
+  alignItems('items-center'),
+  space('space-y-2'),
+  padding('px-8'),
+  gap('gap-2')
+)
+
+const letsGoButton = classnames(
+  display('flex'),
+  justifyContent('justify-center'),
+  alignItems('items-center'),
+  gap('gap-x-3'),
+  fontSize('text-lg'),
+  backgroundColor('bg-night'),
+  borderRadius('rounded-full'),
+  padding('py-4', 'px-6'),
+  width('w-full'),
+  opacity('disabled:opacity-70')
+)
+const errorText = classnames(
+  textColor('text-red-500'),
+  textAlign('text-center')
+)
+
+export default function () {
+  useEffect(() => {
+    const handleMessage = async (message: unknown) => {
+      if (typeof message !== 'object' || !message || !('data' in message))
+        return
+      const { data } = message as { data: string }
```

Comment:
You'll not need `as { data: string }` if you'll create a helper that checks the type